scss/_settings_colors.scss

```diff
@@ -100,8 +100,8 @@ $colors--light-theme--text-inactive: rgba($color-x-dark, $inactive-text-opacity-
 $colors--light-theme--background-default: #fff !default;
 $colors--light-theme--background-alt: #f7f7f7 !default;
 $colors--light-theme--background-inputs: adjust-color($color-x-dark, $lightness: 100% * (1 - $input-background-opacity-amount)) !default;
-$colors--light-theme--background-active: adjust-color($color-x-dark, $lightness: 100% * (1 - $active-background-opacity-amount)) !default;
-$colors--light-theme--background-hover: adjust-color($color-x-dark, $lightness: 100% * (1 - $hover-background-opacity-amount)) !default;
+$colors--light-theme--background-active: hsl(220, 55%, 94%) !default;
```

Comment:
It would probably be good to express it with color variable rather than hardcoding, but I don't know enough about colours math.

We would need some help from @lyubomir-popov here.

Reply:
@bartaz Camille worked on the states, maybe talk to her?